service/docs/source/geopmsession.1.rst

```diff
@@ -27,6 +80,31 @@ core zero:
     $ echo "MSR::THERM_STATUS# core 0" | geopmsession
     0x0000000088430800
 
+This will execute one read of the signal.
+
+Reading a signal periodically
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Both a polling period and runtime must be specified.
```

Comment:
If both must be specified, then we should show both in the `Read a signal at a specific period` synopsis.